src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -263,5 +268,47 @@ public string CreateAuthorizationHeader()
                 _authenticationScheme?.AuthorizationHeaderPrefix ?? TokenType,
                 AccessToken);
         }
+
+#if !NETSTANDARD1_3
+        /// <summary>
+        /// Creates an object to hold the claims from an ID Token.
+        /// </summary>
+        /// <returns>ClaimsPrincipal object containing the claims from the ID Token.</returns>
+        public ClaimsPrincipal GetIdTokenClaims()
```

Comment:
We should probably name the method `GetUser()`. This would be consistent with what .NET, ASP.NET, and ASP.NET Core do?

Reply:
I took a look at the api review doc: https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FAccountDomainModel%2FOverview.md&_a=preview&version=GBdev&anchor=account-%26-tenant-profiles. Here the method is named account.getClaims(). But it is not exposed in authenticationResult  directly.